extension/_test/footnote.txt

```diff
@@ -66,3 +66,26 @@ test![^1]
 </ol>
 </section>
 //= = = = = = = = = = = = = = = = = = = = = = = =//
+6
+//- - - - - - - - -//
+test named[^named].
+
+test incorrect index number[^42]
+
+[^named]: footnote
+[^42]: footnote
+//- - - - - - - - -//
+<p>test named<sup id="fnref:named"><a href="#fn:named" class="footnote-ref" role="doc-noteref">named</a></sup>.</p>
```

Comment:
I've tested this example with [official markdown extra online converter](https://michelf.ca/projects/php-markdown/dingus/), It seem this should be `<sup id="fnref:named"><a href="#fn:named" class="footnote-ref" role="doc-noteref">1</a></sup>`